cmake_converter/main.py

```diff
@@ -103,6 +103,20 @@ def main():  # pragma: no cover
         default=False,
         action='store_true'
     )
+    parser.add_argument(
+        '-pch', '--advanced-precompiled-headers',
+        help='enable advanced custom compatibility function for PCH support',
+        dest='advanced_precompiled_headers',
+        default=False,
+        action='store_true'
+    )
+    parser.add_argument(
```

Comment:
this change is not related to this PR subject. ipo changes should be extracted to another PR.